client/shared/src/hover/actions.ts

```diff
@@ -479,7 +480,74 @@ export function registerHoverContributions({
             })
             subscriptions.add(syncRemoteSubscription(referencesContributionPromise))
 
-            return Promise.all([definitionContributionsPromise, referencesContributionPromise])
+            let implementationsContributionPromise: Promise<unknown> = Promise.resolve()
+            if (!window.context?.enableLegacyExtensions) {
```

Comment:
`window.context` is not set on the browser extensions so this will evaluate to `!undefined` which obviously coerces to `true` so this check will pass on browser extensions 🤔 

I think it would be less risky to not change the browser extension behavior for now:

```suggestion
            if (window.context?.enableLegacyExtensions === false) {
```

Reply:
We actually need to do this because the browser extensions does not load the noop extensions controller so there's going to be a mismatch in behavior if we start registering these random contributions for the legacy extensions controller 🙈